packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -437,13 +437,11 @@ export class DocumentDeltaConnection
         // - a string: log it in the message (if not a string, it may contain PII but will print as [object Object])
         // - a socketError: add it to the OdspError object for driver to be able to parse it and reason
         //   over it.
-        let message = `socket.io: ${handler}`;
-        if (typeof error === "string") {
-            message = `${message}: ${error}`;
-        }
         const errorObj = createGenericNetworkError(
-            message,
+            `socket.io:${handler}`,
```

Comment:
I'd to it reverse - the most useful data is the message, not what kind of handler was hit.
Given that 99% of queries people do in this space would include Data_error, and it takes an effort to even figure out that one needs to look into socketError property, it's safe to say that reverse would be more useful.

Take a look at results of this query:

union Office_Fluid_FluidRuntime_*
| where Data_error contains "socket.io"
| summarize count() by Data_error

Reply:
My goal is to leave `Data_error` with a small set of mostly static values that can direct us back to where in code the error originated / propagated.  Today, `Data_error` is totally diluted due to variable content like timestamps or docIds.  Your query proves the point - it returns 25k rows with count values as has high as 82k.  This is not useful for aggregating to find top errors.

I believe that after this change, `Data_error` + `Data_statusCode` will effectively group all these rows by the actual error case, with the specific details from that session in the `Data_socketError` column.  Here's a query that simulates what queries could look like once this change ships:

<!-- csl: https://kusto.aria.microsoft.com/6a8929bcfc6d44e9b13fee392ada9cf0 -->
```kusto
union Office_Fluid_FluidRuntime_*
| where Data_error has "socket" and Data_error has "connect_document_error" and Data_error has "token"
| summarize count(), dcount(Data_error), any(Data_error) by Data_statusCode
| extend parts = split(any_Data_error, "error:")
| extend Data_error = strcat(parts[0], "error"), Data_socketError = parts[1]
| project Data_error, Data_statusCode, count_, dcount_Data_socketError = dcount_Data_error, any_Data_socketError = Data_socketError
```

|Data_error|Data_statusCode|count_|dcount_Data_socketError|any_Data_socketError|
|---|---|---|---|---|
|socket.io: connect_document_error|400|3129|2453| Token expired. 1622752366.069 > 1622745804.|
|socket.io: connect_document_error|403|23503|21896| Invalid token expiration time. 1624507684000 < 1624507685445.|
|socket.io: connect_document_error|401|167|1| Invalid token type. Unexpected token type "Anonymous".|